src/compile.rs

```diff
@@ -305,6 +305,24 @@ pub fn warn_missing_py_init(artifact: &Path, module_name: &str) -> Result<()> {
                 }
             }
         }
+        goblin::Object::Mach(mach) => {
+            match mach {
+                goblin::mach::Mach::Binary(macho) => {
+                    for sym in macho.exports()? {
+                        let sym_name = sym.name;
+                        if py_init == sym_name.strip_prefix('_').unwrap_or(&sym_name) {
+                            found = true;
+                            break;
+                        }
+                    }
+                }
+                goblin::mach::Mach::Fat(_) => {
+                    // Ignore fat macho,
+                    // we only generate them by combining thin binaries which is handled above
+                    found = true
+                }
+            }
+        }
         _ => {
             // Currently, only linux is implemented
```

Comment:
```suggestion
            // Currently, only linux and mac are implemented
```

Reply:
Also `/// Currently the check is only run on linux` -> `/// Currently checks linux and mac` up where I can't place suggestions